src/db/controller/impl/level.ts

```diff
@@ -23,9 +24,12 @@ export class LevelDbController extends EventEmitter implements IDatabaseControll
 
   private opts: LevelDBOptions;
 
-  public constructor(opts: LevelDBOptions) {
+  private logger: WinstonLogger;
```

Comment:
Can you add a logger interface, `ILogger` and use that everywhere instead of `WinstonLogger`? Then we could swap out loggers in our cli functions if we want later down the line.